docs/sdks/stash/uid2-sdk-ref-mobile-gma.md

```diff
@@ -0,0 +1,99 @@
+# Developer Guide for UID2 GMA Plugin (iOS and Android)
```

Comment:
Where is this page linked from? Should we have a mobile app integration guide page on these mobile sdk/GMA page? Or should we start small and just add on
https://unifiedid.com/docs/overviews/overview-publishers

Under resources section to say for mobile app integration, check the mobile sdk/gma guide?

Reply:
Yes. Also it will be on the integration guides summary page in a Mobile Apps section. You will see that in the PR for GMA doc. For now, this file should not have been in this PR, sorry.